docs/02-providers/aws/README.md

```diff
@@ -41,8 +41,20 @@ provider:
             - AWS::EC2::Instance
 ```
 
-### Deployment S3Bucket
-The bucket must exist beforehand and be in the same region as the deploy.
+### Deployment S3 Bucket
+The bucket must exist beforehand and be in the same region as the deploy. Due to the importance and hard dependency of the deployment bucket, if you want to switch between the core and custom bucket, you have to do the following:
+
+* **From Default Bucket to Custom Bucket:** You'll need to manually empty the default bucket so that it would be removed by CloudFormation and replaced with the custom bucket.
```

Comment:
"You need to manually empty the default bucket that gets created through CloudFormation. On the next deployment the bucket will be removed and we will use the custom bucket you configured. Without emptying the default bucket your next deployment will fail"